src/time.cr

```diff
@@ -398,6 +398,11 @@ struct Time
     days[month]
   end
 
+  # Returns how many days are there in a year
```

Comment:
Can we change description/comment like this?

```
# Returns number of days in *year*
#
# ```
# Time.days_in_year(1990) # => 365
# Time.days_in_year(2004) # => 366
# ```
```

That way, `year` is clearly identified as argument and the example provides a nice touch 😄 

Thank you! ❤️ ❤️ ❤️ 

Ref: https://crystal-lang.org/docs/conventions/documenting_code.html

Reply:
There should be a `.` at the end of the doc sentence.

Reply:
@luislavena thanks for the link. I wasn't aware of that, changing the code to such format.

PS: There are similar methods without such code examples, would a PR with such doc changes be welcomed 😄  would love to contribute like that.